WordPress/Classes/ViewRelated/Site Creation/Shared/TitleSubtitleTextfieldHeader.swift

```diff
@@ -33,8 +33,12 @@ private final class SearchTextField: UITextField {
         return bounds.insetBy(dx: Constants.textInset, dy: 0)
     }
 
-    override func placeholderRect(forBounds bounds: CGRect) -> CGRect {
-        return bounds.insetBy(dx: Constants.textInset, dy: 0)
+    override func textRect(forBounds bounds: CGRect) -> CGRect {
+        let startX = bounds.origin.x + Constants.textInset
+        let startY = bounds.origin.y
+        let width = bounds.width - Constants.textInset * 2
+        let height = bounds.height;
```

Comment:
Thanks for identifying that this should be `textRect(forBounds:)` instead of `placeholderRect(forBounds:)`!

This is _behaving_ correctly, but when debugging, I noticed that the computed rectangle was invalid:
Bounds (input) : (0.0, 0.0, 100.0, 100.0)
Text rect (output) : (56.0, 0.0, -12.0, 100.0)

How would you feel about replacing this with the following?
```
let textInsets = UIEdgeInsets(top: 0, left: Constants.textInset, bottom: 0, right: 0)
return bounds.inset(by: textInsets)
```

Reply:
Again, thank you! I didn't catch that the rect was invalid. Verified yours is so that's perfect! Updated and will merge once the CI approves as well.